src/events/2021-03.md

```diff
@@ -0,0 +1,14 @@
+---
+date: 2021-03-10T19:00:00Z
+description: 
+speakers:
+ - who: Max
+   twitter: omgmog
+   what: The Pico-8 virtual console
+ - who: Mike
+   twitter: 
+   what:  
```

Comment:
Tell us your name, @mikerhyssmith!

Reply:
Hi @benfoxall ! You got my name spot on 😂 

```suggestion
   twitter: mikerhyssmith
   what:  Backstage and onboarding new developers
```

Reply:
Oh man 🤦  I really need some plugin that tells me "You're @benfoxbotica"

Reply:
@benfoxall You're @benfoxbotica